packages/core/src/MPEContract.js

```diff
@@ -62,108 +64,121 @@ class MPEContract {
+    return account.sendTransaction(this.address, channelAddFundsOperation, channelIdStr, amount);
   }
 
   /**
    * Extends an existing payment channel
    * @param {Account} account - The account against which the operations needs to be performed
    * @param {BigNumber} channelId - The payment channel id
-   * @param {BigNumber} expiration - The expiry in terms of block number to extend the channel
+   * @param {BigNumber} expiry - The expiry in terms of block number to extend the channel
    * @returns {Promise.<TransactionReceipt>}
    */
-  async channelExtend(account, channelId, expiration) {
-    logger.info(`Extending PaymentChannel[id: ${channelId}]. New expiry is block# ${expiration}`, { tags: ['MPE'] });
+  async channelExtend(account, channelId, expiry) {
+    const channelIdStr = new BigNumber(channelId).toFixed();
+    const expiryStr = new BigNumber(expiry).toFixed();
+    logger.info(`Extending PaymentChannel[id: ${channelIdStr}]. New expiry is block# ${expiryStr}`, { tags: ['MPE'] });
     const channelExtendOperation = this.contract.methods.channelExtend;
-    return account.sendTransaction(this.address, channelExtendOperation, channelId, expiration);
+    return account.sendTransaction(this.address, channelExtendOperation, channelIdStr, expiryStr);
   }
 
   /**
    * Extends and adds funds to an existing payment channel
    * @param {Account} account - The account against which the operations needs to be performed
    * @param {BigNumber} channelId - The payment channel id
-   * @param {BigNumber} expiration - The expiry in terms of block number to extend the channel
-   * @param {BigNumber} amount - The number of tokens to fund the channel
+   * @param {BigNumber} expiry - The expiry in terms of block number to extend the channel
+   * @param {BigNumber} amountInCogs - The number of tokens to fund the channel
    * @returns {Promise.<TransactionReceipt>}
    */
-  async channelExtendAndAddFunds(account, channelId, expiration, amount) {
-    await this._fundEscrowAccount(account, amount);
+  async channelExtendAndAddFunds(account, channelId, expiry, amountInCogs) {
+    const channelIdStr = new BigNumber(channelId).toFixed();
+    const amount = new BigNumber(amountInCogs).toFixed();
+    const expiryStr = new BigNumber(expiry).toFixed();
+    await this._fundEscrowAccount(account, amountInCogs);
 
-    logger.info(`Extending and Funding PaymentChannel[id: ${channelId}] with amount: ${amount} and expiry: ${expiration}`, { tags: ['MPE'] });
+    logger.info(`Extending and Funding PaymentChannel[id: ${channelIdStr}] with amount: ${amount} and expiry: ${expiryStr}`, { tags: ['MPE'] });
     const channelExtendAndAddFundsOperation = this.contract.methods.channelExtendAndAddFunds;
-    return account.sendTransaction(this.address, channelExtendAndAddFundsOperation, channelId, expiration, amount);
+    return account.sendTransaction(this.address, channelExtendAndAddFundsOperation, channelIdStr, expiryStr, amount);
   }
-
+  
```

Comment:
Remove extra spacing

Reply:
Addressed in [commit](https://github.com/singnet/snet-sdk-js/pull/17/commits/5c2daa487965be1991c316051f66ccc837c9000a)